tools/topology/topology1/platform/intel/intel-generic-dmic.m4

```diff
@@ -46,7 +46,8 @@ dnl Unless explicitly specified, dmic period at 48k is 1ms
 ifdef(`DMIC_48k_PERIOD_US', `', `define(DMIC_48k_PERIOD_US, 1000)')
 
 ## Prolong period to 4ms for RTNR
-ifdef(`RTNR', `define(`INTEL_GENERIC_DMIC_PERIOD', 4000)', `define(`INTEL_GENERIC_DMIC_PERIOD', 1000)')
+ifdef(`RTNR', `define(`INTEL_GENERIC_DMIC_PERIOD', 10000)', `define(`INTEL_GENERIC_DMIC_PERIOD', 1000)')
+ifdef(`RTNR', `define(`DMIC_48k_PERIOD_US', 10000)', `define(`DMIC_48k_PERIOD_US', 1000)')
```

Comment:
Can we keep the RTNR and DMIC period in the top level topology as it is easier to see if there are inconsistencies at those ones.

Reply:
@lgirdwood thoughts on this?

Reply:
ack - simplification is good.